src/main/kotlin/io/github/kryszak/gwatlin/api/gamemechanics/model/skill/SkillSlot.kt

```diff
@@ -10,18 +10,46 @@ import kotlinx.serialization.Serializable
 enum class SkillSlot {
     @SerialName("Downed_[1-4]")
     DOWNED1_4,
```

Comment:
What about previous values? Are they still there, or maybe were they removed in favour of options below?
The same goes for following values (profession, weapon)

Reply:
I searched the JSON files I had and I didn't find any strings ending with literal `[1-5]`. I removed those `SkillSlot` constants.